app/controllers/exchanges/broker_applicants_controller.rb

```diff
@@ -137,4 +136,12 @@ def find_broker_applicant
   def check_hbx_staff_role
     redirect_to exchanges_hbx_profiles_root_path, :flash => { :error => "You must be an HBX staff member" } unless current_user.has_hbx_staff_role?
   end
+
+  def enable_bs4_layout
+    @bs4 = true
+  end
+
+  def resolve_layout
+    EnrollRegistry.feature_enabled?(:bs4_admin_flow) ? "progress" : "single_column"
```

Comment:
is progress right here, or can we use `bootstrap_4` instead?

Reply:
I'm trying to standardize everything to progress for consistency as progress can work for pages with or without a side nav